tests/test_pdf_cruijff.py

```diff
@@ -0,0 +1,91 @@
+"""Tests for Cruijff PDF."""
+
+import numpy as np
+import pytest
+import tensorflow as tf
+import zfit
+from numba_stats import cruijff as cruijff_numba
+from scipy import integrate
+
+# Important, do the imports below
+from zfit.core.testing import tester
+
+import zfit_physics as zphys
+
+# specify globals here. Do NOT add any TensorFlow but just pure python
+mu_true = 90.0
+sigmal_true = 5.0
+alphal_true = 3.0
+sigmar_true = 10.0
+alphar_true = 2.0
+
+
+def create_cruijff(mu, sigmal, alphal, sigmar, alphar, limits):
+    obs = zfit.Space("obs1", limits=limits)
+    cruijff = zphys.pdf.Cruijff(mu=mu, sigmal=sigmal, alphal=alphal, sigmar=sigmar, alphar=alphar, obs=obs, norm=False)
+    return cruijff, obs
+
+
+def test_cruijff_pdf():
+    # Test PDF here
+    cruijff, _ = create_cruijff(
+        mu=mu_true, sigmal=sigmal_true, alphal=alphal_true, sigmar=sigmar_true, alphar=alphar_true, limits=(50, 130)
+    )
+    assert zfit.run(cruijff.pdf(90.0)) == pytest.approx(
+        cruijff_numba.density(
+            90.0,
+            beta_left=alphal_true,
+            beta_right=alphar_true,
+            loc=mu_true,
+            scale_left=sigmal_true,
+            scale_right=sigmar_true,
+        ).item(),
+        rel=1e-8,
+    )
+    np.testing.assert_allclose(
+        cruijff.pdf(tf.range(50.0, 130, 10_000)),
+        cruijff_numba.density(
+            tf.range(50.0, 130, 10_000).numpy(),
+            beta_left=alphal_true,
+            beta_right=alphar_true,
+            loc=mu_true,
+            scale_left=sigmal_true,
+            scale_right=sigmar_true,
+        ),
+        rtol=1e-8,
+    )
+    assert cruijff.pdf(tf.range(50.0, 130, 10_000)) <= cruijff.pdf(90.0)
+
+
+def test_cruihff_integral():
+    # Test CDF and integral here
+    cruijff, obs = create_cruijff(
+        mu=mu_true, sigmal=sigmal_true, alphal=alphal_true, sigmar=sigmar_true, alphar=alphar_true, limits=(50, 130)
+    )
+    full_interval_numeric = zfit.run(cruijff.numeric_integrate(obs, norm_range=False))
```

Comment:
```suggestion
    full_interval_numeric = cruijff.numeric_integrate(obs, norm=False).numpy()
```
Same with zfit.run, also, use `norm` whenever, `norm_range` is the old name